modules/pytket-qiskit/tests/qiskit_convert_test.py

```diff
@@ -682,3 +682,13 @@ def test_convert_multi_c_reg() -> None:
     qcirc = tk_to_qiskit(c)
     circ = qiskit_to_tk(qcirc)
     assert circ.get_commands()[0].args == [m0, q1]
+
+
+# test that tk_to_qiskit works after adding OpType.CRx and OpType.CRy (TKET-2386)
```

Comment:
Can you remove the internal Jira ticket reference?

Reply:
Ah good point, added another commit for that.